src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -76,6 +78,9 @@ public TracerSettings(IConfigurationSource source)
                                false;
 
             Integrations = new IntegrationSettingsCollection(source);
+
+            GlobalTags = source?.GetDictionary(ConfigurationKeys.GlobalTags) as IDictionary<string, string> ??
```

Comment:
`GetDictionary()` already returns the correct type. No need to cast.
```suggestion
            GlobalTags = source?.GetDictionary(ConfigurationKeys.GlobalTags) ??
```